assets/js/builder/Views/Question.js

```diff
@@ -84,7 +84,8 @@ define( [
 		/**
 		 * Compiles the template and renders the view
 		 * @return   self (for chaining)
-		 * @since    3.16.0
+		 * @since 3.16.0
+		 * @since [version] What I did.
```

Comment:
really you left what I did?
:D

Reply:
wait, I know this. I've updated it. Gonna push the code in sometime

Reply:
:D what do you mean?

Reply:
haha! my fault, I just committed the suggested changes. I'm going to push changes in a few minutes (only one is left to do)

Reply:
fixed it. thank you for your patience :p